make.py

```diff
@@ -409,47 +410,56 @@ def coverage():
           f"results in {FULL_BUILD_PREFIX}/coverage/python.txt.")
 
 
-@app.command()
+@app.command(rich_help_panel="Docs")
 def xml():
     run_doc([SPHINXBUILD, "-b", "xml", *ALLSPHINXOPTS, f"{BUILDDIR}/xml"])
     print()
     print(f"Build finished. The XML files are in {FULL_BUILD_PREFIX}/xml.")
 
 
-@app.command()
+@app.command(rich_help_panel="Docs")
 def pseudoxml():
     run_doc([SPHINXBUILD, "-b", "pseudoxml", *ALLSPHINXOPTS, f"{BUILDDIR}/pseudoxml"])
     print()
     print(f"Build finished. The pseudo-XML files are in {FULL_BUILD_PREFIX}/pseudoxml.")
 
 
-@app.command()
+@app.command(rich_help_panel="Coding")
 def lint():
-    run([RUFF, *RUFFOPTS])
-    print("Ruff Finished.")
-    run([MYPY, *MYPYOPTS])
-    print("Mypy Finished.")
+    """
+    Run all linting tasks: ruff and mypy
```

Comment:
If you can squeeze it in, I would add a nudge for users to run this before making a PR.